src/nodes/wishart.jl

```diff
@@ -13,8 +13,8 @@ import Distributions: Wishart
     m_q_out = mean(q_out)
 
     return 0.5 * (
-        m_q_ν * (logdet(m_q_S) + d * log(2)) -
-        logdet(m_q_out) * (m_q_ν - d - 1.0) +
-        tr(cholinv(m_q_S) * m_q_out) + 0.5 * d * (d - 1) * logπ
+        m_q_ν * (mean(logdet, q_S) + d * log(2)) -
+        mean(logdet, q_out) * (m_q_ν - d - 1.0) +
+        tr(mean(cholinv, q_S) * m_q_out) + 0.5 * d * (d - 1) * logπ
     ) + mapreduce(i -> loggamma(0.5 * (m_q_ν + 1.0 - i)), +, 1:d)
```

Comment:
Are we allowed to do this? I see that it is the case in ForneyLab. So basically we compute `E_q(v)[lnGamma(v/2 - (j-1)/2)]` if I am not mistaken. Can we state `E_q(v)[lnGamma(v/2 - (j-1)/2)] = lnGamma(E_q(v)[v]/2 - (j-1)/2)`?